### ISSUE TITLE
Supress activity stream notification

### ISSUE THREAD

**Issue by Luticus** [nextcloud/twofactor_totp#196]
I would like to request an addition to the activity toggles in the personal settings for the suppression of the "You successfully logged in using two-factor authentication (TOTP (Authenticator app))".  I can't really think of a reason to have that on, at all, honestly.  Maybe it could be inverted to show failed log in attempts instead.  Would be a lot less spammy and more useful that way and would show security breach attempts rather than "everything is probably normal" indicators.  Regardless though, I think it would still be a good idea to give users the ability to suppress the messages.

**Comment by xXSTrikeXx** [comment]
Good Idea. The user is going to activitys to see whats going on and not so see what he already knows, namely his recent login :D

**Comment by ChristophWurst** [comment]
> Good Idea. The user is going to activitys to see whats going on and not so see what he already knows, namely his recent login :D

LOL this feature was meant to let you know about any login so that you also see when others logged into your account 🙈

**Comment by ChristophWurst** [comment]
Anybody who's interested in fixing this, these are the relevant lines: https://github.com/nextcloud/twofactor_totp/blob/f56d4422926822e2b43edbfb3e41c836ee15ce45/lib/Activity/Setting.php#L57-L63

**Comment by xXSTrikeXx** [comment]
I thought in the personal settings under recent activitys are the latest logins? I overthinked it and with TOPT the login is only spamming for weblogin. So it makes sense, because with an App password its only connects to the files and not the settings. Anyway I just miss settings. Maybe we can run a discussion on irc :) Just write me ;)

**Comment by ChristophWurst** [comment]
> Maybe we can run a discussion on irc :) Just write me ;)

No need for a discussion. I'm fine with having this configurable. If you want it that way feel free to fix it, push it as pull request and I'll happily review and integrate it in the next release :)

**Comment by ChristophWurst** [comment]
FYI that would make an easy PR to earn a free tshirt https://hacktoberfest.digitalocean.com/ 😉

**Comment by xXSTrikeXx** [comment]
There is just the idea to set it up in personal setting like that how it is:
https://user-images.githubusercontent.com/245432/30855594-cdf7edb0-a2b5-11e7-840a-4fe0e2091292.png
I cloned the master repo now and it´s not listed in "E-Mail" and "Stream". The public functions isdefaultEnabled confused me, because I thought it´s implementing it already. 💃 
When you tell me how I can add a line to the activity app then i could make i try to open a pr. But atm I´m really noobish in programming and I´m willing to learn it, but I haven´t got an overview of nextcloud ;) :D. 

And after the text there could be a checkbox like "only show failed TOPT attempts" or sth like that ;) I think this will help @Luticus
